src/test/java/com/thealgorithms/misc/MatrixTransposeTest.java

```diff
@@ -0,0 +1,58 @@
+package com.thealgorithms.misc;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class MatrixTransposeTest {
+
+    @Test
```

Comment:
To avoid code duplication, these tests should be refactored to `@ParameterizedTest`

Reply:
Done!